pandas/core/window.py

```diff
@@ -511,6 +511,7 @@ class Window(_Window):
         Set the labels at the center of the window.
     win_type : str, default None
         Provide a window type. If ``None``, all points are evenly weighted.
+        Other types are only applicable for `mean` and `sum` functions.
```

Comment:
this *should* be on .agg so I wouldn't add anything here

Reply:
`agg` is using `shared_docs`. I was avoiding to copy it and make a single line change. Will remove it only if no objection